test/isManager.test.js

```diff
@@ -1,14 +1,26 @@
 const assert = require('assert');
-const zoo = require('../src/zoo');
+const { isManager, employeeManagement} = require('../src/isManager');
 
-describe('Implemente a função isManager', () => {
-  it('Testa se o id passado é de um gerente', () => {
-    let actual = zoo.isManager('c5b83cb3-a451-49e2-ac45-ff3f54fbe7e1');
-    let expected = false;
-    assert.deepStrictEqual(actual, expected);
+describe('4 - Crie uma função que faça a gestão das pessoas funcionárias de acordo com a respectiva pessoa gerente', () => {
+  describe('Implemente a função isManager', () => {
+    it('Testa se o id passado é de um gerente', () => {
+      const actual = isManager('c5b83cb3-a451-49e2-ac45-ff3f54fbe7e1');
+      const expected = false;
+      assert.deepStrictEqual(actual, expected);
+    });
+  });
+
+  describe('Implemente a função employeeManagement', () => {
+    it('retorna o nome das pessoas funcionárias do respectivo gerente', () => {
```

Comment:
Como no primeiro 'it' você utilizou a primeira letra maiúscula, não faria sentido usar aqui e no pŕoximo it também?
```suggestion
    it('Retorna o nome das pessoas funcionárias do respectivo gerente', () => {
```